packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -142,6 +142,21 @@ export class AccountResolver {
     return !!result;
   }
 
+  @Mutation((returns) => Boolean)
+  public async recordAdminSecurityEvent(
+    @Args('uid') uid: string,
+    @Args('name') name: string
+  ) {
+    // the ipAddr and ipHmacKey values here are required, but also have no bearing on this type of record.
+    // the securityEvents table is being repurposed to store a broader variety of events, hence the dummy values.
+    await this.db.securityEvents.create({
+      uid,
+      name: 'emails.clearBounces',
+      ipAddr: '0',
+      ipHmacKey: '0',
```

Comment:
We should probably find a way to address the security scan here instead of ignoring it. Seems like we can side bar on this as there are a few ideas floating around.

Reply:
Turns out the auth-server has a config setting for the hmac key. Replicating that approach here is probably the best way to handle this.

Reply:
I took a stab at this -- let me know if it looks sensible to you!